src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6893,6 +6893,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_NotEnoughCloseBracesForRawString" xml:space="preserve">
     <value>The interpolation must end with the same number of closing braces as the number of '$' characters that the raw string literal started with.</value>
   </data>
+  <data name="ERR_CannotMixVerbatimAndRawStrings" xml:space="preserve">
+    <value>Raw string literals cannot be verbatim ('@'). If needed, increase the number of quote characters starting and ending the literal</value>
```

Comment:
Could you add a period to the second sentence for consistency?

Reply:
done.